### PR TITLE
cmd/scollector: fix for "The description for Event ID" in event logs

### PR DESCRIPTION
**Pull request by gbrayut**
See http://stackoverflow.com/questions/29130586
This will update the EventMessageFile to have the correct value when installing or running the service

### CODE REVIEW

File: cmd/scollector/service_windows.go [hunk 1]
```diff
@@ -71,7 +73,9 @@ func installService(name, desc string) error {
 		return err
 	}
 	defer s.Close()
-	err = eventlog.InstallAsEventCreate(name, eventlog.Error|eventlog.Warning|eventlog.Info)
+	// Workaround for bug in eventlog.InstallAsEventCreate
```
Comment (mjibson) [inline cmd/scollector/service_windows.go]: What exactly is the bug? Comparing this to the implementation of InstallAsEventCreate ([here](https://code.google.com/p/winsvc/source/browse/eventlog/install.go#61)), it is identical to your change except that this one calls ExpandEnv and the other one doesn't. Can you describe the bug more?

**Comment by gbrayut** [comment]
Add patch from upstream https://code.google.com/p/winsvc/issues/detail?id=12

Also the registry key stuff could be swapped out with a call to reg.exe if we really don't want to do that in native go

File: cmd/scollector/service_windows.go (hunk 1 above)
Comment (gbrayut) [inline cmd/scollector/service_windows.go]: This workaround is no longer needed, as I included the upstream patch. We still need to fix any existing installs that are currently broken by manually expanding the %systemroot% variable in the REG_SZ key. Ping me when you have time to jump on a hangout if you want more details.

**Comment by mjibson** [comment]
LGTM. Merge when you like.

**Pull request merged**